minemeld/ft/auscert.py

```diff
@@ -50,71 +46,33 @@ def _load_side_config(self):
             LOG.error('%s - Error loading side config: %s', self.name, str(e))
             return
 
-        self.username = sconfig.get('username', None)
-        if self.username is not None:
-            LOG.info('%s - username set', self.name)
-
-        self.password = sconfig.get('password', None)
-        if self.password is not None:
-            LOG.info('%s - password set', self.name)
+        self.api_key = sconfig.get('api_key', None)
+        if self.api_key is not None:
+            LOG.info('%s - api_key set', self.name)
 
     def _build_iterator(self, now):
-        if self.username is None or self.password is None:
```

Comment:
This check should be kept but modified to check for *api_key* field. The exception thrown is propagated to the WebUI to tell the user that the API key should be specified.

Something like:
```
if self.api_key is None:
    raise RuntimErorr(...)
```